spec/models/service_ansible_playbook_spec.rb

```diff
@@ -0,0 +1,92 @@
+describe(ServiceAnsiblePlaybook) do
+  let(:tower_job)      { FactoryGirl.create(:ansible_tower_job) }
+  let(:tower_job_temp) { FactoryGirl.create(:ansible_configuration_script) }
+  let(:basic_service)  { FactoryGirl.create(:service_ansible_playbook, :options => dialog_options) }
+
+  let(:loaded_service) do
+    service_template = FactoryGirl.create(:service_template)
+    service_template.resource_actions.build(:action => 'Provision', :configuration_template => tower_job_temp)
+    service_template.save!
+    FactoryGirl.create(:service_ansible_playbook, :options => provision_options, :service_template => service_template)
+  end
+
+  let(:executed_service) do
+    basic_service.tap do |service|
+      allow(service).to receive(:job).with('Provision').and_return(tower_job)
+    end
+  end
+
+  let(:dialog_options) do
+    {
+      :dialog => {
+        'dialog_hosts'      => 'host1;host2',
```

Comment:
We have not yet handled hosts, but I created a shared document for discussion and recording.